src/feed-input.scss

```diff
@@ -0,0 +1,86 @@
+@import "./new-settings";
+@import "./mixins";
+
+$block: #{$fd-namespace}-feed-input;
+
+.#{$block} {
+  $fd-feed-input-min-height: 3rem;
+  $fd-feed-input-max-height: 20rem;
+  $fd-feed-input-max-width: calc(100vw * 2 / 3);
+  $fd-feed-input-padding-start: 1rem;
+  $fd-feed-input-padding-end: 0.5rem;
+  $fd-feed-input-thumb-margin-start: 0;
+  $fd-feed-input-thumb-margin-end: 0.5rem;
+  $fd-feed-input-btn-margin-start: 0.5rem;
+  $fd-feed-input-btn-margin-end: 0;
+
+  @include fd-reset();
+
+  display: flex;
+  flex-direction: row;
+  padding-top: 1rem;
+  padding-bottom: 1rem;
+  padding-left: $fd-feed-input-padding-start;
+  padding-right: $fd-feed-input-padding-end;
+  background-color: var(--sapField_Background);
+
+  @include fd-screen(m) {
+    margin-top: 1rem;
+    margin-bottom: 1rem;
+  }
+
+  @include fd-screen(l) {
+    max-width: $fd-feed-input-max-width;
+  }
+
+  @include fd-rtl() {
+    padding-right: $fd-feed-input-padding-start;
+    padding-left: $fd-feed-input-padding-end;
+  }
+
+  &__thumb {
+    display: none;
+    flex: 1 0 auto;
+    margin-top: 0;
+    margin-bottom: 0;
```

Comment:
Aren't those values set to 0 by `fd-reset`?

Reply:
removed it